.buildkite/pipeline-raw.yml

```diff
@@ -159,10 +170,10 @@ steps:
    commands:
    - eval "$SET_VERSION"
    - buildkite-agent artifact download "docker/octez-*" . --step build-via-docker
-   - nix develop .#docker-tezos-packages -c ./docker/build/ubuntu/build.py --os ubuntu --type binary --binaries-dir docker --distributions focal
+   - nix develop .#docker-tezos-packages -c ./docker/build/ubuntu/build.py --type binary --binaries-dir docker --distributions focal
```

Comment:
I get this change, but it's unrelated to the `only_changes' ones (and it's not strictly necessary either)

So, please move it to its own separate `[Chore]` commit.